pkgs/applications/video/dvb-apps/default.nix

```diff
@@ -1,14 +1,21 @@
-{ stdenv, fetchurl, perl }:
+{ stdenv, fetchurl, perl, pkgsi686Linux }:
 
-stdenv.mkDerivation {
-  name = "dvb-apps-7f68f9c8d311";
+pkgsi686Linux.stdenv.mkDerivation {
+  name = "dvb-apps-3d43b280298c";
 
   src = fetchurl {
-    url = "https://linuxtv.org/hg/dvb-apps/archive/7f68f9c8d311.tar.gz";
-    sha256 = "0a6c5jjq6ad98bj0r954l3n7zjb2syw9m19jksg06z4zg1z8yg82";
+    url = "https://www.linuxtv.org/hg/dvb-apps/archive/3d43b280298c.tar.bz2";
+    sha256 = "0mz02mz4j945c53vdwaxrpy6fks19nnn482jhg72pqyppq72z7pk";
   };
 
-  buildInputs = [ perl ];
+  buildInputs = [ perl pkgsi686Linux.glibc.static ];
+
+  patches = [
```

Comment:
Please use `fetchpatch` to pull the patches via URL instead of adding them nixpkgs.